litmus-portal/frontend/src/components/CustomCard/styles.ts

```diff
@@ -0,0 +1,143 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    height: '20rem',
+    background: theme.palette.background.paper,
+    boxShadow: theme.shadows[4],
+    borderRadius: 8,
+    overflow: 'hidden',
+    fontSize: 14,
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+    display: 'flex',
+    flexDirection: 'column',
+    flexWrap: 'wrap',
+    textAlign: 'center',
+    cursor: 'pointer',
+    // Above tablet size
+    [theme.breakpoints.up('md')]: {
+      width: 250,
+      margin: theme.spacing(2),
+    },
+  },
+  // CUSTOM WORKFLOW CARD
+  customCard: {
+    background: theme.palette.grey[200],
+    boxShadow: theme.shadows[4],
+    borderRadius: 8,
+    overflow: 'hidden',
+    fontSize: 14,
+    margin: '0 auto',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    flexWrap: 'wrap',
+    textAlign: 'center',
+    cursor: 'pointer',
+    // Above tablet size
+    [theme.breakpoints.up('md')]: {
+      width: 250,
+      margin: theme.spacing(2),
+    },
+  },
+  customWorkflowContent: {
+    margin: '1rem auto',
+    fontWeight: 900,
```

Comment:
Can we use theme.spacing for margin where it isn't in the given format?

Reply:
The changes were not reflection when I was using `margin: `${theme.spacing(0.5)} auto`` but 1 rem is a pretty small change and this being static won't be an issue I think.

Reply:
And it would add further 3 lines to the code if he uses that as its different for top/bottom and left/right. I think '1rem auto' is a better option here. @Saranya-jena